cbdr/cortex
# COR-697: 500s on Cortex All Posts Feed Response

**Purpose:**
When the consumer team hits our post endpoint Redis gives them a 500 timeout error because there are just too many posts that they're querying. The intent of this PR is to alleviate that by creating a new endpoint that return everything *without* hitting Redis.

> The consumer team is currently getting 500’s when making a request to the Article Feed API. Ask Alex for the email chain!

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/COR-697

**Changes:**
* Changes to setup
  * N/A

* Architectural changes
  * N/A

* Migrations
  * N/A
  
* Library changes
  * N/A

* Side effects
  * N/A

**Screenshots**
* Before
N/A

* After
N/A

**QA Links:**
C66 is still down

**How to Verify These Changes**
* Specific pages to visit
  * API /feed/all-posts

* Steps to take
  * See that everything is there and it didn't crash

* Responsive considerations
  * N/A

**Relevant PRs/Dependencies:**
N/A

**Additional Information**
N/A
